satpy/readers/fci_l1c_nc.py

```diff
@@ -441,7 +446,7 @@ def calibrate(self, data, key):
         if key['calibration'] == "counts":
             # from package description, this just means not applying add_offset
             # and scale_factor
-            data.attrs["units"] = "1"
+            pass
         elif key['calibration'] in ['brightness_temperature', 'reflectance', 'radiance']:
             data = self.calibrate_counts_to_physical_quantity(data, key)
         else:
```

Comment:
How about
```python
        if key['calibration'] in ['brightness_temperature', 'reflectance', 'radiance']:
            data = self.calibrate_counts_to_physical_quantity(data, key)
        elif key['calibration'] != "counts":
...
```

Reply:
Yeah, that would work too. I kept the `pass` just to make clear that we don't do any calibration operation, but of course it's an unnecessary `if` branch in the end. I'll simplify the code as suggested.